dotnet.cmd

```diff
@@ -21,7 +21,7 @@ if NOT [%ERRORLEVEL%]==[0] (
 
 pushd %~dp0
 echo Running: dotnet %*
-call "%~dp0\Tools\dotnetcli\dotnet.exe" %*
+powershell -ExecutionPolicy ByPass -NoProfile -command "& """%~dp0eng\dotnet.ps1""" %*"
```

Comment:
Switched to powershell here.  
+ Needed to use powershell to source the tools.ps1 script.  
+ Needed arcade's dotnet path
+ Not sure how to get the arcade dotnet path when it is added to the powershell environment.

This required most of the escaping changes.

Reply:
Would switching from dotnet.cmd to dotnet.ps1 help with the escaping?

Reply:
I think build.cmd et. al would need to change to ps1